packages/plugins/src/foundry/decode.ts

```diff
@@ -280,11 +257,18 @@ export const makeParsedConfig = (
     unlabeledContracts.push(...unlabeledAdditionalContracts)
 
     if (isRawCreate2ActionInput(input)) {
-      const fullyQualifiedName = findFullyQualifiedName(
+      const fullyQualifiedName = findFullyQualifiedNameForInitCode(
```

Comment:
Renamed this function to avoid an ambiguous name due to the presence of a new function, `findFullyQualifiedNameForInitCode`